airbyte-integrations/connectors/source-jira/integration_tests/full_configured_catalog.json

```diff
@@ -7455,10 +7455,10 @@
         },
         "supported_sync_modes": ["incremental"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["created"]
+        "default_cursor_field": ["fields", "updated"]
```

Comment:
I dont sure about this change.
Previously issues.fields where a field containing additionalproperties. Yes, updated exists there. 
It were stored as JSON column, and it will be possibly returned to additionalproperties without details (like it was before), to avoid data missing and db structure change. We'll discuss this tomorrow